Sources/Turf/Geometries/Polygon.swift

```diff
@@ -297,4 +297,55 @@ extension Polygon {
             ring[2].longitude == ring[0].longitude
         )
     }
-}
+
+
+    /// Calculates the centroid using the mean of all vertices.
+    /// This lessens the effect of small islands and artifacts when calculating the centroid of a set of polygons.
```

Comment:
I see you’ve added permalinks to the JavaScript tests that you ported; can you add a similar permalink to the implementation that you ported too? That way we can easily update the implementation if anything changes in the JavaScript version.

Reply:
Sure thing. Done.